packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -137,9 +137,12 @@ class _NetworkScreenBodyState extends State<NetworkScreenBody>
   Widget build(BuildContext context) {
     return Column(
       children: [
-        _NetworkProfilerControls(
-          controller: controller,
-          offline: offlineDataController.showingOfflineData.value,
+        OfflineAwareControls(
+          controlsBuilder: (_) => _NetworkProfilerControls(
+            controller: controller,
+            offline: offlineDataController.showingOfflineData.value,
```

Comment:
use the value from the `controlsBuilder` instead of looking this up from the notifier again:

```
controlsBuilder: (offline) => _NetworkProfilerControls(
  controller: controller,
  offline: offline,
),
```

Reply:
done